charts/ingress-nginx/Chart.yaml

```diff
@@ -15,10 +15,13 @@ keywords:
   - nginx
 kubeVersion: '>=1.21.0-0'
 maintainers:
+- name: cpanato
 - name: Gacko
+- name: puerco
 - name: rikatz
 - name: strongjz
 - name: tao12345666333
+- name: ubergesundheit
```

Comment:
Why add this user? I did not see that he made enough contributions.

Reply:
He's a maintainer for the Helm chart for a while, see `OWNERS_ALIASES`, and I'd like to keep them in sync.

Reply:
True! I see myself more on the reviewer side of things. No need to include me in the list.

Reply:
Fixed